app/Http/Controllers/Accounts/AccountController.php

```
@@ -162,4 +162,41 @@ public function getCashflowOfMonth($accountIds, $before_month = 0)
 
         return $cashflow;
     }
+
+    /**
+     * Update sync options for an account.
+     */
+    public function updateSyncOptions(Request $request, $id)
```

PHPStan (2.1.15)

Method App\Http\Controllers\Accounts\AccountController::updateSyncOptions() has no return type specified.


(missingType.return)